include/kernels/gauge_utils.cuh

```diff
@@ -19,6 +19,7 @@ namespace quda
   // matrix+matrix = 18 floating-point ops
   // => Total number of floating point ops per function call
   // dims * (2*18 + 4*198) = dims*828
+  using computeStapleOps = thread_array<int, 4>;
```

Comment:
Not a blocker, but if this is being defined here, we should use this type down on line 29

Reply:
I disagree with your logic here @weinbe2: it's not a case that we're using this type in this kernel function, rather this user defined type is set to match the type used in the kernel function.  Making line 29 use `computeStapleOps` would serve to obfuscate the code.